.github/workflows/update-coverage-badge.yml

```
@@ -0,0 +1,82 @@
+        node-version: '18'
+        cache: 'npm'
+
+    - name: Install dependencies
+      run: npm ci
+
+    - name: Run tests with coverage
+      run: npm test -- --coverage --silent
+
+    - name: Extract coverage percentage
+      id: coverage
+      run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2002:style:1:16: Useless cat. Consider 'cmd < file | ..' or 'cmd file | ..' instead

(shellcheck)